src/DiagnosticsProvider.php

```diff
@@ -79,6 +79,17 @@ public static function getDiagnostics($node) : \Generator {
                     }
                 }
             }
+            elseif ($node instanceof Node\CatchClause) {
+                $lastElement = \end($node->qualifiedNameList->children);
+                if($lastElement instanceof Token && $lastElement->kind == TokenKind::BarToken) {
```

Comment:
nit: space between if and paren here and everywhere else. sorry, we don't have automatic code style checking yet #83